org.eclipse.jgit/src/org/eclipse/jgit/lib/CommitBuilder.java

```diff
@@ -316,6 +340,13 @@ public Charset getEncoding() {
 			w.flush();
 			os.write('\n');
 
+			if (getGpgSig() != null) {
+				os.write(hgpgsig);
+				os.write(' ');
+				os.write(getGpgSig());
+				os.write('\n');
```

Comment:
Is this the official JGit way of writing headers? Shouldn't there be a utility/helper method?